packages/edit-post/src/store/index.js

```diff
@@ -10,15 +10,18 @@ import reducer from './reducer';
 import applyMiddlewares from './middlewares';
 import * as actions from './actions';
 import * as selectors from './selectors';
+import controls from './controls';
+import { STORE_KEY } from './constants';
 
-const store = registerStore( 'core/edit-post', {
+const store = registerStore( STORE_KEY, {
 	reducer,
 	actions,
 	selectors,
+	controls,
 	persist: [ 'preferences' ],
 } );
 
 applyMiddlewares( store );
-store.dispatch( { type: 'INIT' } );
+store.dispatch( actions.init() );
```

Comment:
Is there any challenge in moving this dispatch to be as part of the `initialize` function in `edit-post/src/index.js`?

Reply:
I don't know.  Certainly could try it.  I only did it here to preserve existing behaviour as much as possible.  The refactors are already a bit of a chore without considerations for other improvements that could be made.  So one the goals with these refactors are:

- preserve existing behaviour as much as possible
- only make changes if it's necessary to keep things working or if there are clear obvious non-complex benefits to doing so.

Certainly, it would make sense to do the dispatch on initialize though, so I could give it a go (especially since its exposed as a action whereas before `INIT` was only a side effect encapsulated in the store - hence why it was probably here).